ksql-rest-model/src/main/java/io/confluent/ksql/rest/entity/KsqlRequest.java

```diff
@@ -124,10 +125,21 @@ public String toString() {
 
   private static Object coerceType(final String key, final Object value) {
```

Comment:
MIght be nice to rename this coerceProperty or similar? It took me a while to figure out what it's doing, which would have been more obvious with a better name